apps/condo/pages/auth/change-password.tsx

```diff
@@ -43,38 +47,66 @@ const ChangePasswordPage: AuthPage = () => {
     const ErrorToFormFieldMsgMapping = {}
 
     const userId = get(auth, ['user', 'id'])
-
+    const authLayoutContext = useContext(AuthLayoutContext)
     const onFinish = (values: typeof initialValues) => {
-        setIsLoading(true)
+        setIsSaving(true)
         const { token, password } = values
         return runMutation({
             mutation: changePassword,
             variables: { data: { token, password } },
-            onFinally: () => {
-                setIsLoading(false)
-                // if (userId) {
-                //     Router.push('/organizations/')
-                // } else {
-                //     Router.push('/auth/signin/')
-                // }
+            onCompleted: ({ data: { result } }) => {
+                const credentials = {
+                    identity: result.email, secret: form.getFieldValue('password'),
+                }
+                reAuth(credentials)
             },
             intl,
             form,
             ErrorToFormFieldMsgMapping,
         }).catch(error => {
-            setIsLoading(false)
+            setIsSaving(false)
+        })
+    }
+    const [signinByEmailMutation] = useMutation(SIGNIN_MUTATION)
+    async function reAuth (credentials: {
+        identity: string;
+        secret: string;
+    }){
+        // await authLayoutContext.signInByEmail(credentials)
+        // auth.signin(credentials.email, credentials.password)
+        await runMutation({
```

Comment:
what method for sign-in should I use here?

Reply:
You can fix  authLayoutContext.signInByEmail(credentials) or remove it